packages/analytics/add-global-link-handler/index.ts

```diff
@@ -44,52 +44,46 @@ export function addGlobalLinkHandler(
 
   window.addEventListener('click', (event) => {
     const linkElement = (event.target as HTMLElement).closest('a')
+    const href = linkElement && linkElement.getAttribute('href')
+    if (!href || !containsDestination(href)) return
```

Comment:
using `linkElement.getAttribute` instead of `.attributes.getNamedItem('href')!
          .value` since it automatically returns the value or null if there is no value

now we just do a null check and the same check as before but inverted